From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Thu, 21 Apr 2016 08:27:09 +0200
Subject: append_kde

---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c49ee41..e16931c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -75,7 +75,6 @@ install(PROGRAMS
   cvs-clean
   cvsblame
   colorsvn
-  svn-clean
   svnlastchange
   svnversions
   extractrc
@@ -96,3 +95,5 @@ install(PROGRAMS
   kde-systemsettings-tree.py
   reviewboard-am
   DESTINATION ${BIN_INSTALL_DIR})
+
+install(PROGRAMS svn-clean DESTINATION ${BIN_INSTALL_DIR} RENAME svn-clean-kde)
