From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Thu, 21 Apr 2016 02:27:13 +0200
Subject: append_kde

---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

Index: kde-dev-scripts/CMakeLists.txt
===================================================================
--- kde-dev-scripts.orig/CMakeLists.txt
+++ kde-dev-scripts/CMakeLists.txt
@@ -83,7 +83,6 @@ install(PROGRAMS
   cvs-clean
   cvsblame
   colorsvn
-  svn-clean
   svnlastchange
   svnversions
   extractrc
@@ -106,4 +105,6 @@ install(PROGRAMS
   grantlee_strings_extractor.py
   DESTINATION ${BIN_INSTALL_DIR})
 
+install(PROGRAMS svn-clean DESTINATION ${BIN_INSTALL_DIR} RENAME svn-clean-kde)
+
 feature_summary(WHAT ALL FATAL_ON_MISSING_REQUIRED_PACKAGES)
